workloads/router-perf-v2/common.sh

```diff
@@ -183,7 +184,12 @@ gen_mb_mix_config(){
       local scheme=https
       local port=443
     fi
-    while read host; do
+    if [[ ${first_termination} == "true" ]]; then
+      first_termination=false
+    else
+      echo ","
```

Comment:
The comma is generated in line 197. I don't understand why you didn't get it generated

Reply:
@rsevilla87 Do you mean why I added the "," in  line 190? If yes, please check the my problem description of `Problem: Misssing ',' between each 'mix_termination'` in the https://github.com/cloud-bulldozer/e2e-benchmarking/pull/201#issue-686565734.  You can see `problem http-scale-mix.json` shows the missing ',' between the last host object of `http` and the first host object of `https`, in short, between 2 'terminations'. While the comma in line 197 generated the ',' between 2 host objects within the `same` termination.